.github/workflows/pypi-package.yml

```diff
@@ -0,0 +1,26 @@
+name: Building & Publishing
+
+on:
+  push:
+    tags: 
+    - "v*"
+    - '!*dev*'
+    - '!*pre*'
+    - '!*post*'
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+
+  # Release to PyPI after a RELEASE gets PUBLISHED in @spacetelescope/astrocut/main
+  publish-to-pypi:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@v1
```

Comment:
we're not defining `upload_to_pypi` anywhere which I believe needs to be set to `true` or a conditional that evaluates to `true` in order for this to actually publish it to pypi

https://github.com/OpenAstronomy/github-actions-workflows/blob/31c0e47a060402a9bd2d97863ea50577d509c050/.github/workflows/publish_pure_python.yml#L126C13-L136

Reply:
I removed that arg because it defaults to `/refs/tags/v` according to their code, which is what I wanted:

https://github.com/OpenAstronomy/github-actions-workflows/blob/31c0e47a060402a9bd2d97863ea50577d509c050/.github/workflows/publish_pure_python.yml#L34

Which is the same as feeding it a `${{ startsWith: github.ref, /refs/tags/v }}` (wrong syntax but you get the point):

https://github-actions-workflows.openastronomy.org/en/stable/publish_pure_python.html